src/main/java/bio/terra/controller/RepositoryApiController.java

```diff
@@ -96,7 +96,8 @@ protected void initBinder(final WebDataBinder binder) {
 
     public ResponseEntity<StudySummaryModel> createStudy(@Valid @RequestBody StudyRequestModel studyRequest) {
         FlightMap flightMap = new FlightMap();
-        flightMap.put("request", studyRequest);
+        flightMap.put(JobMapKeys.REQUEST.getKeyName(), studyRequest);
+        flightMap.put(JobMapKeys.DESCRIPTION.getKeyName(), "Creating a study");
```

Comment:
I'd suggest making the description something that the UI user would like to see. Maybe include the name of the study being created?